resources/src/event_handlers/message_event_handler.rs

```diff
@@ -0,0 +1,32 @@
+use crate::commands;
+use serde_json::Value;
+
+pub async fn handle_message_event(body: &Value) {
+    // Destructure everything needed
+    let is_bot: bool = body["event"]["subtype"] == "bot_message";
+    if is_bot {
+        log::info!("This is a bot");
+        return;
+    }
+    let channel: &str = body["event"]["channel"]
+        .as_str()
+        .unwrap_or("invalid_channel");
+    let text: String = body["event"]["text"]
+        .as_str()
+        .unwrap_or("invalid_text")
+        .to_lowercase();
+    let timestamp: &str = body["event"]["event_ts"]
+        .as_str()
+        .unwrap_or("invalid_timestamp");
+    let enterprise_user_id: &str = body["enterprise_id"].as_str().unwrap_or("invalid_channel");
```

Comment:
We should create a constants file for common constants like this. Or have enums. There's too much room for human error typing some of these fields incorrectly (or wasted time from devs not knowing what fields exist when adding new functionality).

Reply:
I imagine this will change as we use slack morphism. My reasoning behind splitting these into event handlers and then deconstructing the body was because the body changes as the event type changes but each event type has a similar body message. I have also considered making structs out of these but I am hoping Slack Morphism will provide most of that for us